pkg/entities/user_submitted_ad.go

```diff
@@ -0,0 +1,114 @@
+package entities
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/defipod/mochi/pkg/model"
+	"github.com/defipod/mochi/pkg/request"
+	"gorm.io/gorm"
```

Comment:
Should separate 3rd package import from internal package import 
```suggestion
	"github.com/bwmarrin/discordgo"
	"gorm.io/gorm"
	
	"github.com/defipod/mochi/pkg/model"
	"github.com/defipod/mochi/pkg/request"
```